ansible/roles/operator-pipeline/templates/openshift/tasks/show-support-link-with-pr.yml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: show-support-link-with-pr
+spec:
+  params:
+    - name: env
+      description: Enviroment. One of [dev, qa, stage, production]
+    - name: type
+      description: Connect case type.
+      default: "CERT"
+    - name: source
+      description: Connect case source.
+      default: "tekton"
+    - name: cert_project_id
+      description: Pyxis certification project id.
+    - name: cert_project_type
+      description: Connect cert project type. Should be Operator Bundle Image for operator pipelines.
+      default: "Operator Bundle Image"
+    - name: pull_request_url
+      default: ""
+  results:
+    - name: comment
+  steps:
+    - name: show-support-link-with-pr
+      image: quay.io/redhat-isv/operator-pipelines-images
+      script: |
+        #! /usr/bin/env bash
+        set -e
+
+        LINK_ENV="$(params.env)"
+        # url-encode parameters
+        LINK_TYPE=$(printf %s "$(params.type)" | jq -sRr @uri)
+        LINK_SOURCE=$(printf %s "$(params.source)" | jq -sRr @uri)
+        CERT_PROJECT_ID=$(printf %s "$(params.cert_project_id)" | jq -sRr @uri)
+        CERT_PROJECT_TYPE=$(printf %s "$(params.cert_project_type)" | jq -sRr @uri)
+        PULL_REUQEST_URL=$(printf %s "$(params.pull_request_url)" | jq -sRr @uri)
+
+        # select the correct connect instance
+        URL_ENV="connect"
```

Comment:
The connect url is already configured in the `set-env` task, you can grab it from the results instead of trying to come up with it here.

Reply:
Updated!